packages/subgraph/abis/PoolMetadataRegistry.json

```diff
@@ -1,54 +1,1958 @@
-[
-  {
-    "inputs": [
-      { "internalType": "contract IVault", "name": "vault", "type": "address" }
-    ],
-    "stateMutability": "nonpayable",
-    "type": "constructor"
+{
+  "abi": [
```

Comment:
I'm not sure this is what you want. I think you probably want just what's nested under this `abi` key for this subgraph